lib/Factorial.js

```diff
@@ -26,13 +26,18 @@ class Factorial extends Expression
     {
         scope = defaultScope.buildDefaultScope(scope);
 
-        this.value = 1;
-
         this.content = this.content.eval(scope, depth + 1);
 
+        if(this.content.value > 170 || this.content.value < 0) 
```

Comment:
So, I didn't actually know that factorial had to be under 170, I had to google it. Please put a comment here just explaining what otherwise seems like a magic number.

Reply:
Done.